From b6538f6e9cc956959494aff0eeade3a0b5733103 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Fri, 10 Jun 2022 14:20:51 +0100
Subject: [PATCH] Install 50-default.rules in /usr/share

Same rationale as https://gitlab.freedesktop.org/polkit/polkit/-/merge_requests/11
and https://gitlab.freedesktop.org/polkit/polkit/-/merge_requests/72 /etc is for
local admin changes, upstream/vendor config files should go in /usr/share
---
 src/polkitbackend/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/polkitbackend/meson.build b/src/polkitbackend/meson.build
index 7c5d443..c35e6c2 100644
--- a/src/polkitbackend/meson.build
+++ b/src/polkitbackend/meson.build
@@ -61,7 +61,7 @@ libpolkit_backend = static_library(
 
 install_data(
   '50-default.rules',
-  install_dir: pk_pkgsysconfdir / 'rules.d',
+  install_dir: pk_pkgdatadir / 'rules.d',
 )
 
 program = 'polkitd'
-- 
GitLab

